modules/software-extensibility/pages/bonita-repository-access.adoc

```diff
@@ -0,0 +1,131 @@
+= Access to Bonita Repository Manager
+:page-aliases: ROOT:bonita-repository-access.adoc
+:description: Setup access to Bonita Repository Manager to download development dependencies.
+
+{description}
```

Comment:
ℹ️ do not forget to mention this is for Enterprise customer only 😄

Reply:
Is `Bonita Repository Manager` the official name ?

Reply:
I will ask Jonathan about it